src/App.jsx

```diff
@@ -6,14 +6,23 @@ import Register from './components/register/register';
 
 import Login from './components/Login';
 import Logout from './components/Logout';
+import ProtectedRoute from './utils/ProtectedRoute';
 
 function App() {
   return (
     <CookiesProvider>
       <Router>
         <Routes>
           <Route exact path="/" element={<Login />} />
-          <Route exact path="/logout" element={<Logout />} />
+          <Route
+            exact
+            path="/logout"
+            element={
+              <ProtectedRoute redirectPath="/">
+                <Logout />
+              </ProtectedRoute>
+            }
+          />
```

Comment:
Please format the `ProtectedRoute` component similar to [here](https://github.com/ctc-uci/waymakers-frontend/blob/master/src/index.js). Looks cleaner :)

Reply:
Here is a method that should work with v6, while keeping the cleaner look of v5: [https://stackoverflow.com/a/70866269](https://stackoverflow.com/a/70866269)